abi/abi.go

```diff
@@ -69,6 +69,10 @@ const (
 	AskGUID = "4ab7b379-bbac-4fe4-a02f-05aef327c782"
 	// ArkGUID is the AMD Root Key GUID
 	ArkGUID = "c0b406a4-a803-4952-9743-3fb6014cd0ae"
+
+	// ExpectedReportVersion is set by the SNP API specification
+	// https://www.amd.com/system/files/TechDocs/56860.pdf
```

Comment:
I'm a bit torn whether to link specific docs into the code base. My concern is that the hyperlink becomes broken and the comment becomes stale. But let's just go ahead and do it this way for now.